### PR TITLE
Work in progress: Fixing broken links

### PR DESCRIPTION
**Pull request by carolynshin**
Replacing a few broken links in the course.

### CODE REVIEW

**Comment by carolynshin** [comment]
@hectorsector can you help me find the correct link for the Quick Reference Guide? 🙏

File: responses/01_make-a-plan.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ If you are moving your project to GitHub as a public project, you may want to co
 
 Mapping users, keeping commit comments, and other data migrations are more complex, but not impossible. For most version control systems there are helpful Open Source <sup>[:book:](https://help.github.com/articles/github-glossary/#open-source)</sup> tools available. Here are a few resources:
 
-  - [GitHub's documentation on importing from other VCS](https://help.github.com/enterprise/2.12/admin/guides/migrations/importing-data-from-third-party-version-control-systems/)
+  - [GitHub's documentation on importing from other VCS](https://help.github.com/en/enterprise/2.18/admin/migrations/importing-data-from-third-party-version-control-systems)
```
[the remark this answers is not in the document]
Comment (hectorsector) [reply responses/01_make-a-plan.md]: If we leave out the version number the URL should automatically redirect to the latest version of GitHub Enterprise Server:

```suggestion
  - [GitHub's documentation on importing from other VCS](https://help.github.com/en/enterprise/admin/migrations/importing-data-from-third-party-version-control-systems)
```

File: responses/01_make-a-plan.md [hunk 2]
```diff
@@ -75,6 +75,9 @@ Mapping users, keeping commit comments, and other data migrations are more compl
 
 Choose the drop-down below that best fits your current situation or for a printable version of the steps in this course, check out the [Quick Reference Guide]({{ host }}/public/{{ course.slug }}.pdf).
```
Comment (hectorsector) [inline responses/01_make-a-plan.md]: I don't think we recreated the quick reference guide when this course changed, @carolynshin. I think we need to remove the link.

**Comment by carolynshin** [comment]
Thanks for the review @hectorsector! Can I go ahead and merge?

**Pull request merged**